lib/foreman_theme_satellite/engine.rb

```diff
@@ -93,6 +93,7 @@ class Engine < ::Rails::Engine
           Katello::Ping.send :include, SatellitePackages
           Katello::Glue::Provider.send :include, DistributorVersion
           Katello::LayoutHelper.send :include, ThemeLayoutHelper
+          Katello::LayoutHelper.send :prepend, ThemeCssClassHelper
```

Comment:
I am never sure whether to use `:prepend` or `:include`. Both work, the whole block here is using `:include`, the other loading of the helper below uses `:prepend`.